tests/streams/java/src/main/java/betterproto/Tests.java

```diff
@@ -0,0 +1,98 @@
+package betterproto;
+
+import com.google.protobuf.CodedInputStream;
+import com.google.protobuf.CodedOutputStream;
+import betterproto.nested.NestedOuterClass;
+import betterproto.oneof.Oneof;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Tests {
+    String path;
+    public Tests(String path) {
+        this.path = path;
+    }
+
```

Comment:
The formatting seems broken here. Should we add a Java formatter to the project?

Reply:
I have added a pre-commit hook to format the Java source files and it seems to work well. I am somewhat concerned that it might not work in the upstream CI, as I believe it relies on running a JAR file and I don't know for sure if the CI will support that, but if it doesn't I can just change that in the upstream PR.

Reply:
Though I'm not sure what you mean by the formatting being broken here, it seems fine to me?

Reply:
If you look at this line in https://github.com/JoshuaLeivers/python-betterproto/commit/efe7567881a17ab94adfb22107142f439bbf6ca3#diff-ed86e18d2005963d279cd63977b8243815f7b1523cb87c1b9fa98edb83a12432R17
There were two empty lines here instead of one

Reply:
Ah, I see, I thought you meant within the constructor itself